scripts/install.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+# Draft CLI installer
+#
+# ######                                 #####  #       ### 
+# #     # #####    ##   ###### #####    #     # #        #  
+# #     # #    #  #  #  #        #      #       #        #  
+# #     # #    # #    # #####    #      #       #        #  
+# #     # #####  ###### #        #      #       #        #  
+# #     # #   #  #    # #        #      #     # #        #  
+# ######  #    # #    # #        #       #####  ####### ###    
+#                                                               
+# usage: 
+#    curl -fsSL https://raw.githubusercontent.com/Tatsinnit/draft/feature/draft-installer/scripts/install.sh | sh
```

Comment:
prior to merging this PR we should update this URL to reflect the Azure/draft repo